src/index.html

```diff
@@ -2,15 +2,22 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-bar" action="#"
+    method="get"  data-qa="big">
+      <input class="search-bar__input search-bar__big" type="text"
+    data-qa="keypress" placeholder="Try &quot;Los Angeles&quot;">
+    </form>
```

Comment:
```suggestion
    <form
      class="search-bar"
      action="#"
      method="get"  
      data-qa="big"
    >
      <input
        class="search-bar__input search-bar__big"
        type="text"
        data-qa="keypress"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```

Reply:
fix formatting, every attribute should be on new line if you started moving it. Move it when you have more than 2 attributes

Reply:
fix everywhere